cmake/versions.json

```diff
@@ -2,10 +2,10 @@
   "packages" : {
     "legate_core" : {
       "version": "23.03.00",
-      "git_url" : "https://github.com/nv-legate/legate.core.git",
+      "git_url" : "https://github.com/magnatelee/legate.core.git",
```

Comment:
Did you mean to point to a commit on your personal fork? Maybe miscommited file?

Reply:
Yes, I did. That's what we're supposed to do to make CI fetch the right branch until the corresponding legate core PR is merged. After the core PR merge, we should change the commit hash back to one in the mainline.

Reply:
here's the core PR that this change needs: https://github.com/nv-legate/legate.core/pull/690